docs/resources/Application.md

```diff
@@ -94,3 +96,7 @@
 | ----------- | ---------------- | ---------------------------------------------------------------------------------------------------------- |
 | scopes      | array of strings | the [scopes](#DOCS_TOPICS_OAUTH2/shared-resources-oauth2-scopes) to add the application to the server with |
 | permissions | string           | the [permissions](#DOCS_TOPICS_PERMISSIONS) to request for the bot role                                    |
+
+## Get Current Bot Application Information % GET /applications/@me
```

Comment:
Hey there, what is going to happen to the [`/oauth2/applications/@me`](https://discord.com/developers/docs/topics/oauth2#get-current-bot-application-information) route? Is it still going to be supported, or is it going to be deprecated and eventually replaced with this route?

Reply:
Nothing changes here. It was just not documented.